### PR TITLE
Skip peer discovery in HyperG

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**

### CODE REVIEW

File: golem/resource/hyperdrive/resourcesmanager.py [hunk 1]
```diff
@@ -85,3 +90,56 @@ def _wrap_resource(self, resource, task_id=None):
...
+            self._peers[node.key] = peer
+
+
+class HyperdrivePeerManager(HyperDriveMetadataManager):
+
+    def __init__(self, daemon_address):
+        super(HyperdrivePeerManager, self).__init__(daemon_address)
+        self._tasks = dict()
+
+    def add(self, task_id, key_id):
```
Comment (Wolf480pl) [inline golem/resource/hyperdrive/resourcesmanager.py:124]: Is this method user anywhere?
Is there any other place where you are adding stuff to `self._tasks`?

File: golem/task/taskserver.py [hunk 2]
```diff
@@ -27,7 +27,70 @@
 tmp_cycler = itertools.cycle(list(range(550)))
 
 
-class TaskServer(PendingConnectionsServer):
+class TaskResourcesMixin(object):
+
+    def add_resource_peer(self, node_name, addr, port, key_id, node_info):
```
Comment (Wolf480pl) [inline golem/task/taskserver.py:34]: Is this called only from dead code?

File: golem/task/tasksession.py [hunk 3]
```diff
@@ -581,7 +583,7 @@ def _react_to_task_result_hash(self, msg):
         secret = msg.secret
         multihash = msg.multihash
         subtask_id = msg.subtask_id
-        client_options = msg.options
```
Comment (Wolf480pl) [inline golem/task/tasksession.py]: We're making the protocol incompatible with 0.8, right? So can we remove `options` from this message, if it's not used anywhere?

**Review by Wolf480pl — CHANGES_REQUESTED** [review]
To me it looks like the per-task_id list of peers will always be empty, because `HyperdrivePeerManager.add(...)` appears to be never called.

File: golem/task/taskserver.py (hunk 2 above)
Comment (mfranciszkiewicz) [reply golem/task/taskserver.py:34]: It's been moved from TaskServer, I'm not removing dead code here

File: golem/resource/hyperdrive/resourcesmanager.py (hunk 1 above)
Comment (mfranciszkiewicz) [reply golem/resource/hyperdrive/resourcesmanager.py:124]: Not *yet*

Comment (Wolf480pl) [reply golem/resource/hyperdrive/resourcesmanager.py:124]: Oh, the requestor adds its own address in `get_for_task` and sends the list to the provider. So it always pulls from requestor. Ok.

File: golem/task/taskserver.py (hunk 2 above)
Comment (Wolf480pl) [reply golem/task/taskserver.py:34]: mmk

**Review by Wolf480pl — APPROVED** [review]

**Pull request merged**